cmd/image-builder/config.go

```diff
@@ -255,6 +255,18 @@ func loadGithubActionsGitState() (GitStateConfig, error) {
 			JobType:         "postsubmit",
 			BaseCommitSHA:   *payload.HeadCommit.ID,
 		}, nil
+	case "workflow_dispatch":
+		var payload github.WorkflowDispatchEvent
+		err = json.Unmarshal(data, &payload)
+		if err != nil {
+			return GitStateConfig{}, fmt.Errorf("failed to parse event payload: %s", err)
+		}
+		return GitStateConfig{
+			RepositoryName:  *payload.Repo.Name,
+			RepositoryOwner: *payload.Repo.Owner.Login,
+			JobType:         "postsubmit",
```

Comment:
We should use a dedicated job type for workflow dispatch.

Reply:
Changed to ondemand value.